server/app/middlewares/auth.js

```diff
@@ -0,0 +1,60 @@
+import validator from 'email-validator';
+import jwt from 'jsonwebtoken';
+
+const validate = (req, res, next) => {
```

Comment:
You might want to separate the authentication middleware from the validation middleware.